query/common/src/main/java/org/codice/ddf/admin/common/fields/base/ListFieldImpl.java

```diff
@@ -114,16 +118,52 @@ public void setValue(List values) {
     public List<Message> validate() {
         List<Message> validationMsgs = super.validate();
 
-        if (validationMsgs.isEmpty()) {
+        if (validationMsgs.isEmpty() && (getList() != null)) {
             List<Message> fieldValidationMsgs = getList().stream()
                     .map(field -> (List<Message>) field.validate())
                     .flatMap(Collection::stream)
                     .collect(Collectors.toList());
-
-            fieldValidationMsgs.forEach(msg -> msg.addSubpath(fieldName()));
             validationMsgs.addAll(fieldValidationMsgs);
         }
 
         return validationMsgs;
     }
+
+    @Override
+    public ListField<T> matchRequired(Field fieldToMatch) {
+        super.matchRequired(fieldToMatch);
+        listFieldType.matchRequired(((ListField)fieldToMatch).getListFieldType());
+
+        for(Field field : getList()) {
+            field.matchRequired(listFieldType);
+        }
+
+        return this;
+    }
+
+    @Override
+    public ListFieldImpl<T> isRequired(boolean required) {
+        super.isRequired(required);
+        return this;
+    }
+
+    @Override
+    public void addToPath(String fieldName) {
+        super.addToPath(fieldName);
+        for(Field field : getList()) {
```

Comment:
you'll need to set the fieldName of each of the fields in this list to the specified index.